test/test_load_cufflinks_fpkm.py

```diff
@@ -0,0 +1,70 @@
+"""
+test_cufflinks : Test that we can correctly load Cufflinks tracking files which
+contain the estimated expression levels of genes and isoforms (computed from
+RNA-Seq reads).
+"""
+
+<<<<<<< HEAD
+from varcode.rna import load_cufflinks_tracking_file
+=======
+from varcode.cufflinks import load_cufflinks_tracking_file
+>>>>>>> de7b6d57bfee1072b528a2e15f94443500485a52
+from nose.tools import eq_
+
+def test_load_cufflinks_genes():
+    genes_df = load_cufflinks_tracking_file("data/genes.fpkm_tracking")
```

Comment:
This should use the `data_path` helper defined in `test/__init__.py` so the tests work regardless of current working directory. See `test/test_read_evidence.py` for an example.